contracts/mocks/ERC20Mock.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+// mock class using ERC20
+contract ERC20Mock is ERC20 {
+  constructor(
```

Comment:
You can actually hardcode the values for name and symbol. This allows people to know what token it is when looking at contract code

Reply:
Is it better to do it in line 14? (`payable ERC20(name, symbol)`), so I don't change the signature of the constructor

Reply:
yes